kuma/wiki/tests/conftest.py

```diff
@@ -312,7 +312,7 @@ def ks_toolbox():
     }
 
     d_json = json.dumps(errors)
-    d_b64 = base64.encodestring(d_json)
+    d_b64 = base64.encodestring(d_json.encode('utf-8'))
```

Comment:
In this case, ``encodestring`` is probably the right thing, since we split on newlines in the next line.

Reply:
I don't understand if there is a change needed here?

Reply:
No, no change needed. I asked you to change encodestring somewhere else, and I didn't want you to change it here.